src/main/resources/static/schema.graphql

```diff
@@ -16,6 +16,7 @@ type Query {
 }
 type Mutation {
     # category
+    addNewCategory(categoryDTO: CategoryDTO!): Category
```

Comment:
Change the name to addCategory.
Is it necessary have ! punctuation, we need to discuss on scrum.